### PR TITLE
Add verified checkmark in notifications

### PR DESCRIPTION
**Pull request by harpreet9629**
There is 1 small thing that is missing, User Verified Checkmark in Notifications.

I added that, Thanks to @AdityaChaudhary0005 for the suggestion :)

Screenshot 2021-07-10 at 20-03-08 Welcome to BitClout

### CODE REVIEW

**Comment by ShhubhamA** [comment]
The  amount of PR you are doing the core devs should pay you lol :)

**Comment by maebeam** [comment]
If you could fix the horizontal alignment I think I like this change. Will ask for input from @lazynina before approving

**Comment by harpreet9629** [comment]
I did vertical alignment top, should I make it to the center?

**Comment by AdityaChaudhary0005** [comment]
Yes, center would be apt

**Comment by harpreet9629** [comment]
Fixed
Screenshot_1

**Comment by harpreet9629** [comment]
> 
> 
> Awesome ! The amount of PR and merge you are doing the core devs should at least verify you and maybe pay you too lol :) 😛

Thank you, for support

File: src/app/notifications-page/notifications-list/notifications-list.component.ts [hunk 1]
```diff
@@ -108,7 +108,10 @@ export class NotificationsListComponent {
       ProfilePic: "/assets/img/default_profile_pic.png",
     };
     const userProfile = this.profileMap[userPublicKeyBase58Check];
-    const actorName = `<b>${actor.Username}</b>`;
+    //const actorName = `<b>${actor.Username}</b>`;
```
Comment (lazynina) [inline src/app/notifications-page/notifications-list/notifications-list.component.ts:111]:
```suggestion
```

File: src/app/notifications-page/notifications-list/notifications-list.component.ts [hunk 2]
```diff
@@ -108,7 +108,10 @@ export class NotificationsListComponent {
...
+    const actorName = (actor.IsVerified) ? `<b>${actor.Username}</b><span class="ml-1 d-inline-block align-center text-primary fs-12px">
```
Comment (lazynina) [inline src/app/notifications-page/notifications-list/notifications-list.component.ts:112]:
```suggestion
    const actorName = actor.IsVerified ? `<b>${actor.Username}</b><span class="ml-1 d-inline-block align-center text-primary fs-12px">
```

**Review by lazynina — CHANGES_REQUESTED** [review]
I like this change - some small code clean up requested before approving this one.

**Comment by harpreet9629** [comment]
All Done

**Review by lazynina — APPROVED** [review]

**Pull request merged**